react/Avatar/InitialsAvatar.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+
+import Avatar, { avatarPropTypes } from '.'
```

Comment:
Should we need to use aliased path on UI too? cc @y-lohse @GoOz

Reply:
Not that I'm aware no.

Reply:
Is that a webpack thingy?